docs/sgp40_voc_index/arduino.rst

```diff
@@ -0,0 +1,110 @@
+SGP40 on Arduino Platforms
+==========================
+
+This guide describes how to get the new sgp40 working on arduino
+platforms. It offers a step by step guide to get basic funcionality working.
+
+Preperation
+-----------
+
+First of all note that the most commenly used Arduino boards have a logic level of 5v but the
```

Comment:
For all instances, Volt uses a capital V symbol. https://en.wikipedia.org/wiki/Volt